.palantir/revapi.yml

```diff
@@ -1018,6 +1018,12 @@ acceptedBreaks:
       old: "method void org.apache.iceberg.PositionDeletesTable.PositionDeletesBatchScan::<init>(org.apache.iceberg.Table,\
         \ org.apache.iceberg.Schema, org.apache.iceberg.TableScanContext)"
       justification: "Removing deprecated code"
+  "1.5.0":
```

Comment:
Looks like adding a new ctor method as suggested changes the serializationVersionUID, is it ok?  @RussellSpitzer

Reply:
+1 Yep, this would only be a concern if we were worried about folks using different Iceberg versions on client and server, this shouldn't be the case.